presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/TransformCorrelatedScalarAggregationToJoin.java

```diff
@@ -94,6 +95,14 @@ public Result apply(CorrelatedJoinNode correlatedJoinNode, Captures captures, Co
             return Result.empty();
         }
 
+        List<ProjectNode> subqueryProjections = searchFrom(correlatedJoinNode.getSubquery(), context.getLookup())
```

Comment:
Can you add a comment about why we avoid this case? (Your explanation above should do).